bundles/net.tourbook/src/net/tourbook/database/PersonManager.java

```diff
@@ -29,117 +30,126 @@
 
 public class PersonManager {
 
-	private static PersonManager			_instance;
+   private static PersonManager           _instance;
 
-	private static ArrayList<TourPerson>	_people	= null;
+   private static ArrayList<TourPerson>   _people = null;
 
-	private static final Object				LOCK	= new Object();
+   private static final Object            LOCK    = new Object();
 
-	public static PersonManager getInstance() {
+   private static ContributionItem_Person _contributionItem_PersonSelector;
 
-		if (_instance != null) {
-			return _instance;
-		}
+   public static ContributionItem_Person get_contributionItem_PersonSelector() {

```

Comment:
Can you rename the `get/set` methods to the same name as in `net.tourbook.application.ApplicationActionBarAdvisor.getPersonSelector()`, remove `_contributionItem_`

Reply:
Fixed